CHANGELOG.md

```diff
@@ -1,6 +1,16 @@
 faucet-pipeline-sass version history
 ==================================
 
+
+v1.10.0
+-------
+
+_TBD_
```

Comment:
This will need to be updated, of course. We might also want to update the Git commit date accordingly.